apps/src/code-studio/components/libraries/LibraryCreationDialog.jsx

```diff
@@ -267,6 +291,12 @@ class LibraryCreationDialog extends React.Component {
       case LoadingState.PUBLISHED:
         bodyContent = this.displaySuccess();
         break;
+      case LoadingState.CODE_ERROR:
+        bodyContent = this.displayCodeError();
```

Comment:
Rather than have two separate functions that do the same thing, can we combine them to something like `displayErrorMessage(msg)` and call them via `bodyContent = this.displayErrorMessage({i18n.libraryCodeError()})`?

Reply:
👍